pkg/models/cedar_deployment.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"github.com/guregu/null/zero"
+)
+
+// CedarDataCenter represents a single DataCenter object returned from the CEDAR API
+type CedarDataCenter struct {
+	ID          zero.String
+	Name        zero.String
+	Version     zero.String
+	Description zero.String
+	State       zero.String // example: "Active" - NOT geographical state
+	Status      zero.String
+	StartDate   zero.Time
+	EndDate     zero.Time
+
+	// address components
+	Address1     zero.String
+	Address2     zero.String
+	City         zero.String
+	AddressState zero.String
+	Zip          zero.String
+}
+
+// CedarDeployment represents a single Deployment object returned from the CEDAR API
+type CedarDeployment struct {
+	// always-present fields
+	ID       string
+	Name     string
+	SystemID string
+
+	// possibly-null fields
+	StartDate                zero.Time
+	EndDate                  zero.Time
+	IsHotSite                zero.String // this could potentially be nullable Bool; sample values from CEDAR seem to be "Yes" or null
```

Comment:
Could also be worth seeing if CEDAR can translate this data to `true`/`false` on their end.

Reply:
Honestly, I think for now it might make the most sense to just leave it to echo the data straight from CEDAR. Don't want this to be held up by their feedback/changes.

Reply:
Sounds good. Comment clarified in https://github.com/CMSgov/easi-app/pull/1406/commits/80bfc997970483040d4fcd708ec54345f7aefa97.